.babelrc

```diff
@@ -1,6 +1,5 @@
 {
   "presets": [
-    "es2015",
```

Comment:
This is no longer the case, you might want to have react-app + use the babel-preset-env